editor/animation_bezier_editor.h

```diff
@@ -178,7 +179,7 @@ class AnimationBezierTrackEdit : public Control {
 	void set_animation_and_track(const Ref<Animation> &p_animation, int p_track);
 	virtual Size2 get_minimum_size() const override;
 
-	void set_undo_redo(UndoRedo *p_undo_redo);
+	void set_undo_redo(Ref<EditorUndoRedoManager> p_undo_redo);
```

Comment:
I'm actually planning to remove all the `set_undo_redo()` methods and just access it directly from the EditorNode singleton (similar to #57306), just not sure if it's better to do it after this PR or before. If done before it might make this PR even bigger.